src/redux/actions/followUnfollowAction.js

```diff
@@ -0,0 +1,89 @@
+import axios from "axios";
+import jwtDecode from "jwt-decode";
+import { toaster } from "evergreen-ui";
+import {
+  FOLLOW_SUCCESS,
+  UNFOLLOW_SUCCESS,
+  NOT_FOLLOWING,
+  FOLLOWING
+} from "../action_types";
+import { serverError } from "./common";
+import config from "../../config";
+
+const baseUrl = config.BASE_URL;
+const token = localStorage.getItem("token");
+
+export const followStatus = author => async dispatch => {
+  const following = () => {
+    dispatch({ type: FOLLOWING });
+  };
+
+  const notFollowing = () => {
+    dispatch({ type: NOT_FOLLOWING });
+  };
+  const allFollowers = await axios.get(
+    `${baseUrl}/profiles/${author}/followers/`,
+    {
+      headers: { Authorization: `Bearer ${token}` }
+    }
+  );
+  const currentUser = jwtDecode(token).username;
+  const checkData =
+    allFollowers.data !== undefined || allFollowers.data.length !== 0;
+  const username = allFollowers.data.find(
+    profile => profile.username === currentUser
+  );
+
+  const ternaryFunc = () => (username ? following() : notFollowing());
+
+  if (checkData) {
+    ternaryFunc();
+  } else {
```

Comment:
Why is the `else` still here

Reply:
When we remove `else`, the ternary operation is not working as expected but we have found a workaround by providing `/* eslint no-unused-expressions: [2, { allowTernary: true }] */`